Integrations/integration-ProtectWise.yml

```diff
@@ -63,96 +98,45 @@ script:
         }
         return token;
     };
-    var downloadEventPcap = function(eventId, filename, token) {
-        var url = params.url+'pcaps/events/'+eventId;
-        if (filename && filename.length > 0 ) {
-            url = url + '?filename=' + filename;
-        }
+
+    var doReq = function(url, token, raw) {
         var res = http(
             url,
             {
                 Headers: {'X-Access-Token': [ token ]},
                 Method: 'GET'
             },
-            params.insecure
+            params.insecure,
+            params.proxy
             );
         if (res.StatusCode !== 200) {
-            throw 'Failed to execute download Event Pcap request:' + res.StatusCode + ', body: ' + res.Body;
+            throw 'Failed to execute ' + url + ' : ' + res.StatusCode + ', body: ' + res.Body;
         }
-        return saveFile(res.Body);
+
+        return (raw ? res.Body : JSON.parse(res.Body));
```

Comment:
You should wrap parse with try catch, so we can return proper error if body was not a proper json
Look at other places you do parse as well